app/policies/maestrano/connector/rails/organization_policy.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Maestrano::Connector::Rails::OrganizationPolicy < Maestrano::Connector::Rails::ApplicationPolicy
+  def create?
```

Comment:
As discussed @manu-d, could be worth to group the policies by strategy (User/Organization and IdMap/Synchronization) and specify them in the models instead? 
```ruby
class class Organization
  def self.policy_class
    CreateOnlyPolicy
  end
end
```